src/main/java/games/loveletter/actions/BaronAction.java

```diff
@@ -2,65 +2,63 @@
 
 import core.AbstractGameState;
 import core.CoreConstants;
-import core.actions.AbstractAction;
 import core.components.Deck;
+import core.components.PartialObservableDeck;
 import core.interfaces.IPrintable;
 import games.loveletter.LoveLetterGameState;
 import games.loveletter.cards.LoveLetterCard;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Objects;
 
-
 /**
  * The Baron lets two players compare their hand card. The player with the lesser valued card is removed from the game.
  */
-public class BaronAction extends core.actions.DrawCard implements IPrintable {
```

Comment:
I'm getting errors when running ForwardModelTester on LoveLetter - and the problem seems to be in the Baron action. Will dig deeper.